.changeset/rotten-lemons-cry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-auth-backend-module-microsoft-provider': minor
```

Comment:
Changeset looks much better now! If a package is still on a `0.y.z` version we try to use `minor` changes only for indicating bigger, breaking changes. Therefor this should be a `patch` change in this case. As soon as we are in the `x.y.z` versions with x >= 1 this would have been a `minor` change (for reference: https://semver.org/spec/v2.0.0.html)

```suggestion
'@backstage/plugin-auth-backend-module-microsoft-provider': patch
```